live-hls.js

```diff
@@ -711,7 +833,17 @@ processErrors = function(request, response, event) {
  * Inject error
  */
 injectError = function(request, response) {
-  var streamname='live' + request.path;
+
+  var playType;
+  for (key in streams) {
```

Comment:
This will have unintended results if there is both an event and live style murphy stream running at the same time

Reply:
But we will have to somehow check if the type is live or event in order to get the correct stream. Also I checked by running both the streams together and it gives the intended results.

Reply:
Right, we do have to check if the type is live or event, however with both an event and live style of the same stream, the stream that gets picked is dependent on the order they are in `streams` and not which stream is actually being requested.

```
streams = {
  'event/maat/planet.m3u8': {},
  'live/maat/planet.m3u8': {}
}
```

Since this loops through the keys in `streams`, the first iteration of the loop will set `playType = 'event'`, but the next iteration will set `playType = 'live'` causing the error to be injected into the `live` type and not the `event` type